gammapy/scripts/image_pipe.py

```diff
@@ -21,37 +28,152 @@ def image_pipe_main(args=None):
+
+    def make_background(self, obs_id):
+        obs = self.data_store.obs(obs_id=obs_id)
+        table = obs.bkg.acceptance_curve_in_energy_band(energy_band=self.energy_band)
+        center = obs.pointing_radec.galactic
+        bkg_hdu = fill_acceptance_image(self.header, center, table["offset"], table["Acceptance"], self.offset_band[1])
+        livetime = obs.observation_live_time_duration
+        self.bkg_image.data = Quantity(bkg_hdu.data, table["Acceptance"].unit) * self.solid_angle * livetime
+        self.bkg_image.data = self.bkg_image.data.decompose()
 
-    def make_background(self):
         log.info('Making background image ...')
 
+    def make_background_normalized(self, obs_id, exclusion_mask):
+        """Normalized the background compare to te events in the counts maps outside the exclusion maps
+
+        Parameters
+        ----------
+        exclusion_mask : `~gammapy.image.ExclusionMask`
+
+        Returns
+        -------
+
+
+        """
+        self.make_counts(obs_id)
+        self.make_background(obs_id)
+        counts_sum = np.sum(self.counts_image.data * exclusion_mask.data)
+        bkg_sum = np.sum(self.bkg_image.data * exclusion_mask.data)
+        scale = counts_sum / bkg_sum
+        self.bkg_image.data = scale * self.bkg_image.data
+
+    def make_total_bkg(self, exclusion_mask):
+        """Stack the total bkg from the observation in the 'DataStore'
+
+        Parameters
+        ----------
+        exclusion_mask : `~gammapy.image.ExclusionMask`
+
+        Returns
+        -------
+
+        """
+        for obs_id in self.data_store.obs_table['OBS_ID']:
+            self.make_background_normalized(obs_id, exclusion_mask)
+            self.total_bkg_image.data += self.bkg_image.data
+
+    def make_images(self, exclusion_mask):
+        """Compute the counts, bkg and exlusion_mask images for a set of observation
+
+        Parameters
+        ----------
+        exclusion_mask : `~gammapy.image.ExclusionMask`
+
+        Returns
+        -------
+        maps : `
+        """
+        self.make_total_counts()
+        self.make_total_bkg(exclusion_mask)
+        maps = SkyMapCollection()
+        maps['counts'] = self.total_counts_image
+        maps['bkg'] = self.total_bkg_image
+        maps['exclusion'] = exclusion_mask
+        return maps
+
+    def make_significance(self, counts_image, bkg_image, radius):
+        """
+
+        Parameters
+        ----------
+        counts_image : ~gammapy.image.SkyMap`
+            count image
+        bkg_image : ~gammapy.image.SkyMap`
+            bkg image
+        radius : float
+        Disk radius in pixels.
+
+        Returns
+        -------
+
+        """
+        counts = disk_correlate(counts_image.data, 10)
+        bkg = disk_correlate(bkg_image.data, 10)
+        s = significance(counts, bkg)
+        s_image = fits.ImageHDU(data=s, header=counts_image.to_image_hdu().header)
+        return s_image
```

Comment:
I'm not sure if the method should return anything at all, or just add additional maps to the `self.maps` attribute (which is a `SkyMapCollection`). The advantage of the later is, that all the available maps are stored in a single container and can be written to file at any time e.g. for debugging